pinot-core/src/main/java/org/apache/pinot/core/realtime/converter/stats/RealtimeColumnStatistics.java

```diff
@@ -139,6 +140,11 @@ public boolean isSorted() {
       return false;
     }
 
+    // Values in BYTES are not comparable
+    if(_block.getMetadata().getDataType().getStoredType() == DataType.BYTES) {
```

Comment:
There's a wrapper around byte[] that is comparable: org.apache.pinot.common.utils.primitiveorg.apache.pinot.common.utils.primitive.ByteArray

Reply:
Update with ByteArray comparison.

Reply:
+1 on using this wrapper comparable. Can we have byte[] column that is sortable?